src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -286,7 +304,15 @@ void ParticipantConfiguration::xmlTagCallback(
           "Please extend the use-mesh tag as follows: <use-mesh name=\"{}\" from=\"(other participant)\" />",
           _participants.back()->getName(), name, name);
     }
-    _participants.back()->useMesh(mesh, offset, false, from, safetyFactor, provide, geoFilter);
+
+    // TODO: Maybe merge later with the config check above
```

Comment:
```suggestion
```
I guess with our current xml implementation we cannot support such a configuration-time check. OK for me like this.